### DISCUSSION TITLE
whisper openai

### DISCUSSION THREAD

**Discussion by Policing2024** [SubtitleEdit/subtitleedit#8858] [Ideas]
how to use whisper commandline with powershell please

**Comment by bme3412** [comment]
1. pip install openai-whisper
2. Install FFmpeg - Whisper requires FFmpeg. 
3. choose the Whisper model: tiny, base, small, medium, large
4. select output format: Available formats include txt, vtt, srt, tsv, json, and all
5. select language (ex English)
6. whisper "C:\Users\YourName\Desktop\my-cool-podcast.mp3" - change filepath to match your device
